pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Pull request checklist:
+ - [ ] Features and bugfixes should be merged to the `develop` branch, **not** `main`
+ - [ ] All text should be internationalized
+ - [ ] There should be no linter errors
+ - [ ] New features should support all states (loading, error, etc) 
```

Comment:
Because this is a checklist, I prefer something more like:
```
- [ ] All text is internationalized
- [ ] There are no linter errors.
- [ ] New features support all states (loading, error, etc) 
```

Reply:
I'm not sure how I would rephrase the first line
```
If this branch is a feature or bugfix, the pull request is into develop, not main
```
?